ciao-launcher/vmconfig.go

```diff
@@ -96,6 +96,14 @@ func (cfg *vmConfig) findVolume(UUID string) *volumeConfig {
 	}
 	return nil
 }
+func (cfg *vmConfig) haveBootableVolume() bool {
```

Comment:
@tpepper @markdryan I actually tried it an it seems to work
`ciao-cli instance add -workload=e35ed972-c46c-4aad-a1e7-ef103ae079a2 -volume uuid=b3d391e9-309d-4271-b76e-1767c44877b8,boot_index=0`
Here the volume was a previously auto cloned volume from another instance.

The only thing that bothers me with this is that we end up with an extra boot volume (the one based off of the image).
This is a bit confusing as, when you boot from a volume, the image definition is immaterial. The bootable volume is the image.

See this
qemu-system-x86_64 
-drive file=rbd:rbd/b3d391e9-309d-4271-b76e-1767c44877b8:id=ciao,if=none,id=drive_b3d391e9-309d-4271-b76e-1767c44877b8,format=raw 
-device virtio-blk-pci,scsi=off,bus=pci.0,addr=0x3,id=device_b3d391e9-309d-4271-b76e-1767c44877b8,drive=drive_b3d391e9-309d-4271-b76e-1767c44877b8 

-drive file=rbd:rbd/2c1bf8a6-6bb6-4201-b463-acaa683d4bdf:id=ciao,if=none,id=drive_2c1bf8a6-6bb6-4201-b463-acaa683d4bdf,format=raw 
-device virtio-blk-pci,scsi=off,bus=pci.0,addr=0x4,id=device_2c1bf8a6-6bb6-4201-b463-acaa683d4bdf,drive=drive_2c1bf8a6-6bb6-4201-b463-acaa683d4bdf 

-drive file=/var/lib/ciao/instances/29ea470d-96f6-4689-afd3-ec120d012c4b/seed.iso,if=virtio,media=cdrom